server/chat/background/prediscovery_task.py

```
@@ -95,7 +95,8 @@ def build_prediscovery_prompt(user_id: str, providers: List[str], integrations:
 ## EXPLORATION STRATEGY
 
 1. **Cloud infrastructure** (if cloud providers connected):
-   - cloud_exec('gcp'/'aws'/'azure', ...) to list clusters, VMs, databases, load balancers
+- cloud_# FIX: 移除exec，改用安全方式
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)